resources/user-guides/mining_with_xmrig_and_docker.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: static_page
+title: "Mining with XMRig and Docker"
+title-pre-kick: "Mining XMR"
+title-kick: "With XMRig and DOcker"
+title-post-kick: ""
+kick-class: "purple-kicks"
+icon: "icon_userguides"
+attribution: "<!-- Icon is based on work by Freepik (http://www.freepik.com) and is licensed under Creative Commons BY 3.0 -->"
+---
+
+##Introduction
```

Comment:
Please add spaces between `##` and the titles to have valid formatted titles. Same applies to the other ones below.